compiler/rustc_typeck/src/check/pat.rs

```diff
@@ -160,7 +160,9 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         ti: TopInfo<'tcx>,
     ) {
         let path_res = match &pat.kind {
-            PatKind::Path(qpath) => Some(self.resolve_ty_and_res_ufcs(qpath, pat.hir_id, pat.span)),
+            PatKind::Path(qpath) => {
+                Some(self.resolve_ty_and_res_fully_qualified_call(qpath, pat.hir_id, pat.span))
+            }
```

Comment:
these diffs appear unnecessary?

Reply:
I believe that is just the method name change + x.py fmt moving it to another line due to the line now being a smidge too long

Reply:
ah, I see, I didn't notice that the method name changed.